tests/lib/pkgdb.sh

```diff
@@ -164,61 +172,67 @@ distro_install_package() {
         ;;
     esac
 
-    for pkg in "$@" ; do
-        package_name=$(distro_name_package "$pkg")
-        # When we could not find a different package name for the distribution
-        # we're running on we try the package name given as last attempt
-        if [ -z "$package_name" ]; then
-            package_name="$pkg"
-        fi
+    set -- $(
```

Comment:
Silly question, but why do `set --` here? Could we also write: `pkg_names=$( for pkg in ...)` ? If so, maybe thats easier to read?

Reply:
`set --` sets `$@`. Certainly it could be done with `pkg_names=($(...))` (note the parens to make it an array), and then the calls to the package managers would be e.g.
    quiet apt-get install $APT_FLAGS -y "${pkg_names[@]}"

is this easier to read than the `set --` and then using `$@`?

Reply:
Thanks, I think I'm convinved